docs/docs/integrations/vectorstores/aerospike.ipynb

```diff
@@ -0,0 +1,706 @@
+    "\n",
+    "When finished, store your AVS instance's IP address and port to use later\n",
+    "in this demo:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "PROXIMUS_HOST = \"<avs-ip>\"\n",
+    "PROXIMUS_PORT = 5000"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Install Dependencies \n",
+    "The `sentence-transformers` dependency is large. This step could take several minutes to complete."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "vscode": {
+     "languageId": "shellscript"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "!pip install --upgrade --quiet aerospike-vector-search==0.6.1 sentence-transformers langchain"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Download Quotes Dataset\n",
+    "\n",
+    "We will download a dataset of approximately 100,000 quotes and use a subset of those quotes for semantic search."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "--2024-05-10 17:28:17--  https://github.com/aerospike/aerospike-vector-search-examples/raw/7dfab0fccca0852a511c6803aba46578729694b5/quote-semantic-search/container-volumes/quote-search/data/quotes.csv.tgz\n",
+      "Resolving github.com (github.com)... 140.82.116.4\n",
+      "Connecting to github.com (github.com)|140.82.116.4|:443... connected.\n",
+      "HTTP request sent, awaiting response... 302 Found\n",
+      "Location: https://raw.githubusercontent.com/aerospike/aerospike-vector-search-examples/7dfab0fccca0852a511c6803aba46578729694b5/quote-semantic-search/container-volumes/quote-search/data/quotes.csv.tgz [following]\n",
+      "--2024-05-10 17:28:17--  https://raw.githubusercontent.com/aerospike/aerospike-vector-search-examples/7dfab0fccca0852a511c6803aba46578729694b5/quote-semantic-search/container-volumes/quote-search/data/quotes.csv.tgz\n",
+      "Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.110.133, 185.199.109.133, 185.199.111.133, ...\n",
+      "Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.110.133|:443... connected.\n",
+      "HTTP request sent, awaiting response... 200 OK\n",
+      "Length: 11597643 (11M) [application/octet-stream]\n",
+      "Saving to: ‘quotes.csv.tgz’\n",
+      "\n",
+      "quotes.csv.tgz      100%[===================>]  11.06M  1.94MB/s    in 6.1s    \n",
+      "\n",
+      "2024-05-10 17:28:23 (1.81 MB/s) - ‘quotes.csv.tgz’ saved [11597643/11597643]\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "!wget https://github.com/aerospike/aerospike-vector-search-examples/raw/7dfab0fccca0852a511c6803aba46578729694b5/quote-semantic-search/container-volumes/quote-search/data/quotes.csv.tgz"
```

Comment:
**Issue**: The wget command is used to download data directly within the Jupyter notebook. Using wget in notebooks can lead to security risks if not properly sanitized and can make the notebook environment dependent. <br> **Fix**: It's safer and more reproducible to include data as part of the repository or to use a Python package like 'requests' to handle downloads. This method allows better error handling and integration within the Python ecosystem. <br> **Code Suggestion**: 
 ```
 - !wget https://github.com/aerospike/aerospike-vector-search-examples/raw/7dfab0fccca0852a511c6803aba46578729694b5/quote-semantic-search/container-volumes/quote-search/data/quotes.csv.tgz
 + import requests
 + url = "https://github.com/aerospike/aerospike-vector-search-examples/raw/7dfab0fccca0852a511c6803aba46578729694b5/quote-semantic-search/container-volumes/quote-search/data/quotes.csv.tgz"
 + r = requests.get(url)
 + with open("quotes.csv.tgz", 'wb') as f:
 +     f.write(r.content)
 ```